crates/oxc_linter/src/snapshots/no_var.snap

```diff
@@ -0,0 +1,349 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 144
+expression: no_var
+---
+  ⚠ eslint-plugin-react(display-ame):
+   ╭─[no_var.tsx:1:1]
+ 1 │ var foo = bar;
+   · ──────────────
```

Comment:
Can we shrink the span to `var`?

Reply:
`decl.start .. decl.start + 3`  should be enough.